### PR TITLE
Added tags support for themes

### PR DESCRIPTION
**Pull request by harrymkt**
**IMPORTANT: Please do not create a Pull Request adding a new feature without discussing it first.**

The place to discuss new features is the forum: <https://zola.discourse.group/>
If you want to add a new feature, please open a thread there first in the feature requests section.

Sanity check:

* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/getzola/zola/pulls) for the same update/change?

## Code changes
(Delete or ignore this section for documentation changes)

* [ ] Are you doing the PR on the `next` branch?

If the change is a new feature or adding to/changing an existing one:

* [x] Have you created/updated the relevant documentation page(s)?

### CODE REVIEW

File: docs/config.toml [hunk 1]
```diff
@@ -5,6 +5,10 @@ description = "Everything you need to make a static site engine in one binary."
 compile_sass = true
 build_search_index = true
 
+taxonomies = [
+  { name = "tags"}
```
Comment (Keats) [inline docs/config.toml]: they should probably be named theme_tag no? In case we ever want other types of tags

**Comment by Keats** [comment]
Can you remove the changes to the processed images?

**Comment by harrymkt** [comment]
Ok, it's resolved now. I previously used the build command to test and that images got processed

File: docs/templates/themes.html [hunk 2]
```diff
@@ -9,6 +9,7 @@
     <div class="themes-container">
         {% block theme_content %}
             <h1>Zola themes</h1>
+            <p><a href="{{ get_url(path = "theme-tags") }}">Search by tags</a></p>
```
Comment (Keats) [inline docs/templates/themes.html]: I think I would wait until we actually have templates with tags before linking to it. No point linking to an empty page.

**Review by Keats — CHANGES_REQUESTED** [review]

File: docs/templates/themes.html (hunk 2 above)
Comment (harrymkt) [reply docs/templates/themes.html]: Zluinav theme has that tags. Meanwhile I'll try finding another way out to see if the link can be invisible if there is no defined content with `theme-tags` taxonomy

**Pull request merged**